cmd/plasmacli/subcmd/query/status.go

```diff
@@ -0,0 +1,32 @@
+package query
+
+import (
+	"fmt"
+	"github.com/FourthState/plasma-mvp-sidechain/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/spf13/cobra"
+)
+
+// StatusCmd returns the current state of the eth connection (syncing, crashed etc)
+func StatusCmd() *cobra.Command {
+	return statuscmd
+}
+
+var statuscmd = &cobra.Command {
+	Use:   "status",
+	Short: "Check current block number",
+	Long: `returns current block number of plasmad connection`,
```

Comment:
s/`/" and s/returns/Returns

Do you think status makes here as the command name? wdyt about "height"? `plasmacli query height`

Reply:
if `status` is used, I think we should maybe print out more info of the connected client

Reply:
Lets go with height for now. Can open up another issue with other metrics we are interested in (and eventually combine). Fixing now